### ISSUE TITLE
Snappyhexmesh error, point is not within Mesh

### ISSUE THREAD

**Issue by Christoffertpaulsen** [ladybug-tools/butterfly#233]
Hello 

I tried create an outdoor wind simulation, for my own 3D model. I have used the example file as given with the installation, and tried an online tutorial by Mostapha. It worked fine with the example, but when I tried to do it on my own, the SnappyHexMesh component showed an error:" 

1. Solution exception:
 --> snappyHexMesh Failed!
--> FOAM FATAL ERROR: 
Point (15.72323 63.68281 10.59472) is not inside the mesh or on a face or edge.
Bounding box of the mesh:(-241 2.334047e+07 0) (69 2.334114e+07 103.5)

    From function static Foam::labelList Foam::refinementParameters::findCells(bool, const Foam::polyMesh&, const pointField&)
    in file snappyHexMeshDriver/refinementParameters/refinementParameters.C at line 206.

FOAM exiting"

I'm not sure what point it is talking about, any idea?﻿

**Comment by Christoffertpaulsen** [comment]
<img width="1381" alt="skaermbillede 2017-05-09 kl 13 05 00" src="https://cloud.githubusercontent.com/assets/28560697/25847867/2a6777ee-34b8-11e7-8969-0f41d756bf7a.png">

**Comment by TheodoreGalanos** [comment]
Hi @Christoffertpaulsen 

OF is quite clear on the error. Your locationInMesh is not currently inside your boundary domain.

To solve this you can connect a boundingbox component to points output of the windTunnel component and then get the center point of that with a volume component. You can then connect the point from the volume component to the locationInMesh input of SHM component.

Also, make sure you are recreating a new case (e.g. adding a new filename) and not just running SHM on a new geometry without adjusting the previous inputs. 

Let us know if that fixes things.

Kind regards,
Theodore.

**Comment by Christoffertpaulsen** [comment]
Hey Theodore

Thank you, it got me a step further! But now when i changed the name to Air_simulation_hjulmagervej I encountered another error, with the component LoadProbesValue: 

1. Solution exception:Failed to load values:
	Failed to find probes folder folder at C:\Users\Christoffer\butterfly\air_simulation_hjulmagervej\postProcessing\probes

I checked if the folder was there, but I couldn't find the postProcessing folder.

**Comment by TheodoreGalanos** [comment]
@Christoffertpaulsen

I have a feeling your probes are under the previous folder name. If the mesh is the same you can move them to the new folder, at the requested location. If the mesh is different simply re-run your case with the new settings.

Kind regards,
Theodore.

**Comment by Christoffertpaulsen** [comment]
Thank you Theodore so far. I really apprieciate it and the program! 
I just seem to not get it, because not it looks like the simulation is running. One minor thing is, that my model disapears when i try to run the component Solutions.  Any idea why this happened?

**Comment by Christoffertpaulsen** [comment]
I actually figured out to simulate on another simple model. I had to change my Brep that went into the GenTestPts component. So that it had the right trimmed surface, matching to the model. But still I had some errors, so I would really like to send you my files, since the model seen in Rhino still disapears in some of the views and the resualt visualization does not work either. 
I hope you can assist me. 

Kind regards
Christoffer

**Comment by mostaphaRoudsari** [comment]
@Christoffertpaulsen can you upload your model here? or send your model to the [grasshopper forum](http://www.grasshopper3d.com/group/ladybug/forum)?

**Comment by Christoffertpaulsen** [comment]
I have uploaded my files to: 
http://www.grasshopper3d.com/group/ladybug/forum/topics/problems-with-butterfly-won-t-simulate-proberly

**Comment by Christoffertpaulsen** [comment]
I actually made it work, thank you!
But it still seems that my 3D model dispears except from the perspective view.
Furthermore I tried to keep on the simulations
![udklip 2](https://cloud.githubusercontent.com/assets/28560697/26029135/fce8a01c-382d-11e7-9452-1c31518a0947.PNG)
 and got something out of it! But I would like the Legend to be visible as well, which I can not. Any solutions to how I can? 
I have attached a screenshot, of how it looks like, when I have set the component to preview. 

Thank you!
